build_and_test.sh

```diff
@@ -1,105 +1,144 @@
 #!/bin/bash
 CURRENTDIR=`pwd`
-CIS_RPC_BUILD_DIR=$CURRENTDIR/build/build-cis-rpc
+RPC_BUILD_DIR=$CURRENTDIR/build/build-rpc
```

Comment:
Will it be appropriate to name build-mem-server or similar name ?

Reply:
Renamed the build directory for memory server model and pushed a new commit.

Regards,
Faizan